var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -188,35 +205,32 @@ def check_install(self):
             # TODO: Automate these path and library settings
             cc('-c', "-I%s" % join_path(spec.prefix, "include"), "check.c")
             cc('-o', "check", "check.o",
-               "-L%s" % join_path(spec.prefix, "lib"),
-               "-L%s" % join_path(spec.prefix, "lib64"),
```

Comment:
@tgamblin  I will try, but since our modification stems from ORNL repository, which I believe far behind, it may not be as simple as that.